core/templates/dev/head/pages/exploration_editor/translation_tab/StateTranslationDirective.js

```diff
@@ -267,7 +267,7 @@ oppia.directive('stateTranslation', [
               StateEditorService.setActiveStateName(
                 ExplorationInitStateNameService.displayed);
             }
-
+            $scope.stateName = StateEditorService.getActiveStateName();
             var stateName = StateEditorService.getActiveStateName();
```

Comment:
Can we remove this stateName then? (as you have populated scope with the state name, so let's use that one instead. will that be fine?)